commerce_coordinator/apps/frontend_app_ecommerce/urls.py

```diff
@@ -4,10 +4,11 @@
 
 from django.urls import include, path
 
-from commerce_coordinator.apps.frontend_app_ecommerce.views import UserOrdersView
+from commerce_coordinator.apps.frontend_app_ecommerce.views import RedirectReceiptView, UserOrdersView
 
 app_name = 'frontend_app_ecommerce'
 urlpatterns = [
     path('order_history/', UserOrdersView.as_view(), name='order_history'),
+    path('order_receipt', RedirectReceiptView.as_view(), name='order_receipt'),
```

Comment:
```suggestion
    path('order_receipt/', RedirectReceiptView.as_view(), name='order_receipt'),
```

Reply:
Why? It works fine without it? and the other project is coded to not have it?

Reply:
Answer: trailing slash is what nginx appends to URLs during routing to the appropriate service. If it is not present, no endpoint would match